sound/attributions.txt

```diff
@@ -28,4 +28,4 @@ splatter.ogg adapted from https://freesound.org/people/Rocktopus/sounds/233418/
 hohoho.ogg and hehe.ogg are cut from a recording by Nanakisan on freesound: https://freesound.org/people/Nanakisan/sounds/253534/
 mbox_full.ogg and mbox_end.ogg make use of The Ragtime Drummer by James Lent, in the public domain
 
-growl1.ogg and growl2.ogg in /sound/creatures/dog are adapted from Glitchedtones's Freesound shih-tzu uploads https://freesound.org/people/Glitchedtones/
+"Bababooey" sound effect is cut from and credited to The Howard Stern Show.
```

Comment:
A: Please do not accidentally delete old stuff credits  
B: Can we uh, legally use this audio?

Reply:
What the fuck I don't remember doing that, hold on.